books/RayTracingInOneWeekend.html

```diff
@@ -1073,29 +1073,29 @@
 the chapter.)
 
 <div class='together'>
-Pick a random point $s$ from the unit radius sphere that is tangent to the hitpoint, and send a ray
-from the hitpoint $\mathbf{p}$ to the random point $\mathbf{S}$. That sphere has center
-$\mathbf{p} + \vec{\mathbf{N}}$:
+Pick a random point s from the unit radius sphere that is tangent to the hitpoint, and send a ray
```

Comment:
This wording is still confusing. The unit radius sphere is tangent to the _surface_, at the hit point `$s$`.

Reply:
Actually, since there are _two_ spheres tangent to the surface, we should further clarify that we want the one on the side that contains the ray origin → sign(Rd ⋅ N) = sign((p - Sc) ⋅ N)

A better diagram would be immensly useful here. I've created #263 to capture this.

Reply:
There are exactly two unit spheres tangent to the surface at hit point $p$ but there are an infinite number of unit spheres tangent to the hit point alone. Wording will need be massaged.